typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumBaseBRETTWarpConfig.ts

```diff
@@ -0,0 +1,40 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  HypTokenRouterConfig,
+  OwnableConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const ISM_CONFIG = ethers.constants.AddressZero; // Default ISM
+
+export const getBaseZeronetworkBrettConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const base: HypTokenRouterConfig = {
+    ...routerConfig.base,
+    ...abacusWorksEnvOwnerConfig.base,
+    type: TokenType.collateral,
+    token: tokens.base.brett,
+    interchainSecurityModule: ISM_CONFIG,
```

Comment:
Should we remove this? I.e. omit ism config if its 0x0?

Reply:
the other configs do this. i'm not sure of the implications of removing it, but also okay with being explicit with this.